cmd/lambda/publish_lease_events/main.go

```diff
@@ -126,6 +126,10 @@ func handleRecord(input *handleRecordInput) error {
 					log.Printf("ERROR: Failed to mark get account %q", lease.AccountID)
 					return err
 				}
+				if acct == nil {
```

Comment:
with functional testing we are occasionally throwing in `nil` values to the queue.  This is because we are truncating the table and not able to query the account.

Reply:
So if we still can't get the account at this point lets assume its gone and we can skip the rest of the work here.